packages/components/inputs/input-utils/src/multiline-input/multiline-input.tsx

```diff
@@ -52,23 +78,4 @@ const MultilineInput = (props) => {
 
 MultilineInput.displayName = 'MultilineInput';
 
-MultilineInput.propTypes = {
-  name: PropTypes.string,
-  className: PropTypes.string,
-  autoComplete: PropTypes.string,
-  id: PropTypes.string,
-  value: PropTypes.string.isRequired,
-  onChange: requiredIf(PropTypes.func, (props) => !props.isReadOnly),
```

Comment:
We might want to add an warning/invariant to cover this case

Reply:
ah good, catch!